sdk/reya_websocket/resources/wallet.py

```
@@ -292,53 +294,28 @@ def unsubscribe(self) -> None:
         self.socket.send_unsubscribe(channel=self.path)
 
 
-class WalletSpotExecutionBustsResource(SubscribableParameterizedResource):
-    """Resource for accessing wallet spot execution busts."""
+class WalletExecutionBustsResource(SubscribableParameterizedResource):
+    """Resource for accessing wallet execution busts (unified spot + perp)."""
 
     def __init__(self, socket: "ReyaSocket"):
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)